mixin.scss

```diff
@@ -0,0 +1,25 @@
+@mixin basekick($bk-type-size-modifier, $bk-type-row-span, $bk-descender-height-scale, $sk-base-font-size, $bk-grid-row-height, $bk-line-height-override: false) {
+  font-size: unit($bk-type-size-modifier * $sk-base-font-size, px);
+
+  @mixin calculate-line-height-as-scale($line-height) {
+    $line-height-scale: ($line-height / ($bk-type-size-modifier * $sk-base-font-size));
+    line-height: unit($line-height-scale, em);
```

Comment:
`unit($line-height-scale, em)` does not seem compatible with SCSS, got it to work using `($line-height-scale * 1em)` but not sure if that is what people do in the SCSS world. Was this working for you?